ons_alpha/topics/models.py

```diff
@@ -63,18 +63,19 @@ class TopicPage(BaseTopicPage):
     subpage_types = ["articles.ArticleSeriesPage", "bulletins.BulletinSeriesPage", "methodologies.MethodologyPage"]
     page_description = "A specific topic page. e.g. Public sector finance or Inflation and price indices"
 
-    def latest_by_series(self, model: Type[Page], series_model: Type[Page]) -> QuerySet:
-        newest = (
-            model.objects.live()
-            .public()
-            .filter(path__startswith=OuterRef("path"), depth__gte=OuterRef("depth"))
-            .order_by("-release_date")
+    def latest_by_series(self, model: Type[Page], series_model: Type[Page], topic: Topic = None) -> QuerySet:
+        newest_qs = (
+            model.objects.live().public().filter(path__startswith=OuterRef("path"), depth__gte=OuterRef("depth"))
         )
+        if topic:
+            newest_qs = newest_qs.filter(topics__topic=topic)
+        newest_qs = newest_qs.order_by("-release_date")
 
+        model_pk_qs = series_model.objects.not_child_of(self) if topic else series_model.objects.child_of(self)
```

Comment:
if a topic is passed, it will look at series pages that are not children of the current topic page.